src/utils/accountUtils.ts

```diff
@@ -129,9 +129,9 @@ export const getJazzSeed = (address: string | undefined) => {
 }
 
 export const formatLargeNumber = (number: BigNumber) => {
-  const BILLION = new BigNumber(1_000_000_000)
-  const MILLION = new BigNumber(1_000_000)
-  const THOUSAND = new BigNumber(1_000)
+  const BILLION = new BigNumber(1000000000)
```

Comment:
Looks like the release bundler doesnt like this format, getting the error:
`error src/utils/accountUtils.ts: Unexpected token name «_000_000_000», expected punc «,» in file src/utils/accountUtils.ts at 119:42.`

Reply:
yeah, not sure who put this in but sadly app center won't allow it.